task-sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -118,6 +118,9 @@ class RuntimeTaskInstance(TaskInstance):
     start_date: datetime
     """Start date of the task instance."""
 
+    is_mapped: bool | None = None
```

Comment:
What does None mean? Can this not be just `is_mapped: bool` without a default, and initiated on creation?

Reply:
i found that when i do what you suggest, mypy complains that task may be None when runtime TI created -- because TI.task is Operator | None. i will cast it but, just fyi

Reply:
ok so i tried it making it no default, and, other stuff broke so, inclined to not deal with that right now 😬

Reply:
Would it make sense if this is `is_mapped: bool = False`?